nixos/modules/services/desktops/espanso.nix

```diff
@@ -8,6 +8,11 @@ in {
   options = {
     services.espanso = {
       enable = mkEnableOption "Espanso";
+      wayland = mkOption {
```

Comment:
You can directly `mkEnableOption` with "using the Wayland compatible espanso package".